microprofile-rest-client/pom.xml

```diff
@@ -192,22 +279,85 @@
+                            <execution>
+                                <goals>
+                                    <goal>package</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+
+        <profile>
+            <id>openshift</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.wildfly.plugins</groupId>
+                        <artifactId>wildfly-maven-plugin</artifactId>
+                        <configuration>
+                            <!-- deploys the quickstart on root web context -->
+                            <name>ROOT.war</name>
+                            <feature-packs>
+                                <feature-pack>
+                                    <location>org.wildfly:wildfly-galleon-pack:${version.server}</location>
+                                </feature-pack>
+                                <feature-pack>
+                                    <location>org.wildfly.cloud:wildfly-cloud-galleon-pack:${version.pack.cloud}</location>
+                                </feature-pack>
+                            </feature-packs>
+                            <layers>
+                                <layer>jaxrs-server</layer>
+                                <layer>microprofile-platform</layer>
+                                <layer>cloud-server</layer>
+                            </layers>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>package</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>set-server-host</id>
+            <activation>
+                <property>
+                    <name>env.SERVER_HOST</name>
+                </property>
+            </activation>
+            <properties>
+                <server.host>${env.SERVER_HOST}</server.host>
+            </properties>
+        </profile>
+        <profile>
+            <id>integration-testing</id>
             <build>
                 <plugins>
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-failsafe-plugin</artifactId>
-                        <version>${version.failsafe.plugin}</version>
+                        <configuration>
+                            <includes>
+                                <include>**/*IT</include>
+                            </includes>
+                            <systemPropertyVariables>
```

Comment:
why not use the all QS shared "standard" way of defining server.host in the test class?

Reply:
Part of the original client code was showing how to use the `microprofile-config.properties`. This had a line like:
```
org.wildfly.quickstarts.microprofile.rest.client.CountriesServiceClient/mp-rest/url=http://localhost:8080/country-server
```

That won't work for on all runtime environments. I changed it to:
```
org.wildfly.quickstarts.microprofile.rest.client.CountriesServiceClient/mp-rest/url=${server.host}
```

It effectively works like the other quickstarts, the configuration is just different.

Reply:
got it thanks